app/services/forecast_service.rb

```diff
@@ -1,16 +1,25 @@
 class ForecastService < BaseService
 
+  def self.base_url
+    "http://api.openweathermap.org"
+  end
+
+  def self.set_api_key(connection)
+    connection.params['key'] = ENV['OW_API_KEY']
+  end
+
   def self.get_city_forecast(city)
     coordinates = MapquestGeoService.get_coordinates(city)
     lat = coordinates[:results][0][:locations][0][:latLng][:lat]
     lon = coordinates[:results][0][:locations][0][:latLng][:lng]
-    response = open_weather_connection.get('/data/2.5/onecall') do |faraday|
-      faraday.params['appid'] = ENV['OW_API_KEY']
-      faraday.params['lon'] = lon
-      faraday.params['lat'] = lat
-      faraday.params['units'] = 'imperial'
+    response = connection.get('/data/2.5/onecall') do |connection|
+      connection.params['appid'] = ENV['OW_API_KEY']
+      connection.params['lon'] = lon
+      connection.params['lat'] = lat
+      connection.params['units'] = 'imperial'
```

Comment:
The Fraday API's various request methods (get, post, etc) all take an optional second argument where you can set params. It's nice because it avoid the block and lets you set them with symbolized keys instead of directly manipulating the `params` object and setting individual string keys on it. I believe this should work:

```
response = connection.get('/data/2.5/onecall', {
  lat: lat,
  lon: lon,
  units: 'imperial'
})
```

(I omitted the API key since that's now handled by our `set_api_key` method.)